scout/utils/scout_requests.py

```diff
@@ -280,10 +301,12 @@ def fetch_exac_constraint():
     exac_lines = None
 
     LOG.info("Fetching ExAC genes")
+    print("Fetching ExAC genes")
```

Comment:
This line is not needed here, you probably used it for debugging

Reply:
Yes thank you